src/include/migraphx/op/gather.hpp

```diff
@@ -38,7 +38,6 @@ struct gather
 
     shape normalize_compute_shape(std::vector<shape> inputs) const
     {
-        check_shapes{inputs, *this}.has(2).standard();
```

Comment:
Shouldn't we still have `check_shapes{inputs, *this}.has(2)`?

Reply:
Yes, you are right. we should keep the check for the number of inputs..